theme-path.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path')
```

Comment:
theme selection logic. here I get the path to theme using the config file.
currently, it uses the folder name to resolve the path to the theme, but this can be done in a different way. which im exploring.